package.json

```diff
@@ -44,7 +44,8 @@
     "graceful-fs": "4.2.4",
     "js-sha3": "^0.8.0",
     "node-localstorage": "^2.1.6",
-    "trezor-connect": "^8.1.19"
+    "trezor-connect": "^8.1.19",
+    "webcrypto": "^0.1.1"
```

Comment:
I meant this https://nodejs.org/api/webcrypto.html the idea is to not add any extra dependency. https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues https://gist.github.com/alexdiliberto/39a4ad0453310d0a69ce